app/models/notification.rb

```diff
@@ -0,0 +1,6 @@
+class Notification < ApplicationRecord
```

Comment:
@FaxriddinMaxmadiyorov launch gem annotate

Reply:
@XaoGao I did it